config/routes.rb

```diff
@@ -27,7 +27,10 @@
     get 'cms', to: 'cms#show'
     namespace :cms do
       resources :collections, except: [:delete, :index]
-      resources :transcripts, except: [:show, :delete, :index]
+      resources :transcripts, except: [:show, :delete, :index] do
+        get "speaker_search", on: :member
```

Comment:
I thought so too, but this raises an error on edit because it expects the member id, this was a work around that allowed me to get the functionality to work.